packages/workflow/src/Interfaces.ts

```diff
@@ -502,6 +510,15 @@ export interface INodeType {
 	};
 }
 
+export interface INodeVersionedType {
+	nodeVersions: {
+		[key: number]: INodeType
+	};
+	defaultVersion: number;
```

Comment:
Ideally only positive integers. Same for the arg in `getNodeType`. I will look into whether TS can enforce this.

Reply:
Not possible at the moment.